samples/boards/nordic/system_off/boards/nrf54l15dk_nrf54l10_cpuapp_retained_mem.overlay

```diff
@@ -0,0 +1,25 @@
+/ {
+	cpuapp_sram@2002e000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x2002e000 DT_SIZE_K(4)>;
+		zephyr,memory-region = "RetainedMem";
+		status = "okay";
+
+		retainedmem0: retainedmem {
+			compatible = "zephyr,retained-ram";
+			status = "okay";
+		};
+	};
+
+	aliases {
+		retainedmemdevice = &retainedmem0;
+	};
+};
+
+&cpuapp_sram {
+	/* Shrink SRAM size to avoid overlap with retained memory region:
+	 * 192 - 4 = 188KB
+	 */
+	reg = <0x20000000 DT_SIZE_K(188)>;
+	ranges = <0x0 0x20000000 0x2e000>;
```

Comment:
This size in `ranges` is wrong. It should be `DT_SIZE_K(188)`. But maybe in this context it would be clearer to use `0x2f000` for both sizes.